word_guess.rb

```diff
@@ -0,0 +1,212 @@
+require 'faker'
+require 'colorize'
+
+# Used 2 classes
+# A 'Word Guess Game' has a 'word'
+
+class WordGuessGame
+  attr_accessor :word, :tries, :lives, :user_guess
+
+  # Word Guess Game has:
+  # nouns: word, levels, lives
+  # & verbs: active play
+
+  def initialize(word)
+    @word = word
+    @tries = tries
+    @lives = ""
+    @user_guess
+  end
+
+  def easy_level
+    @lives = 5
+    @user_guess = ""
+    puts active_play(@user_guess)
+  end
+
+  def hard_level
+    @lives = 3
+    @user_guess = ""
+    puts active_play(@user_guess)
+  end
+
+  def active_play(user_guess)
```

Comment:
when you call the method `active_play`, it doesn't actually do anything with the input so you don't really need the parameter I think.